data/aashish05.json

```diff
@@ -0,0 +1,13 @@
+{
+"name": "Aashish Kumar",
+"irc": "virus",
+"irc_channels": [rust],
+"show_avatar": false,
+"email": "aashish050697@gmail.com",
+"discourse": "Rust",
```

Comment:
This field is intended to be your forum username i.e. the user that you type in when logging into https://users.rust-lang.org/.